<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dobran-O-Viz 0.3</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout" />
    <addaction name="actionHelpHelp" />
    <addaction name="separator" />
    <addaction name="actionHelpEnableTooltips" />
   </widget>
   <widget class="QMenu" name="menuFilters" >
    <property name="title" >
     <string>F&amp;ilters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRecent_Files" >
    <property name="title" >
     <string>Recent Files</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPipeline" >
    <property name="title" >
     <string>Pipeline</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources" >
    <property name="title" >
     <string>Sources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuServer" >
    <property name="title" >
     <string>&amp;Server</string>
    </property>
    <addaction name="actionServerConnect" />
    <addaction name="actionServerDisconnect" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionToolsCreateCustomFilter" />
    <addaction name="actionToolsManageCustomFilters" />
    <addaction name="separator" />
    <addaction name="actionToolsDumpWidgetNames" />
    <addaction name="actionToolsRecordTest" />
    <addaction name="actionToolsRecordTestScreenshot" />
    <addaction name="actionToolsPlayTest" />
    <addaction name="separator" />
    <addaction name="actionToolsPythonShell" />
   </widget>
   <widget class="QMenu" name="menuRecentFiles" >
    <property name="title" >
     <string>Recent Files</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditUndo" />
    <addaction name="actionEditRedo" />
    <addaction name="separator" />
    <addaction name="actionEditCameraUndo" />
    <addaction name="actionEditCameraRedo" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles" >
     <property name="title" >
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionFileNew" />
    <addaction name="actionFileOpen" />
    <addaction name="separator" />
    <addaction name="actionFileLoadServerState" />
    <addaction name="actionFileSaveServerState" />
    <addaction name="separator" />
    <addaction name="actionFileSaveData" />
    <addaction name="actionFileSaveScreenshot" />
    <addaction name="actionFileSaveAnimation" />
    <addaction name="separator" />
    <addaction name="actionFileExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuServer" />
   <addaction name="menuSources" />
   <addaction name="menuFilters" />
   <addaction name="menuPipeline" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="pipelineBrowserDock" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" />
  </widget>
  <widget class="QDockWidget" name="objectInspectorDock" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Object Inspector</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" />
  </widget>
  <widget class="QDockWidget" name="statisticsViewDock" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Statistics View</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4" />
  </widget>
  <widget class="QDockWidget" name="elementInspectorDock" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Element Inspector</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5" />
  </widget>
  <widget class="QToolBar" name="VCRToolbar" >
   <property name="windowTitle" >
    <string>VCR Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionVCRFirstFrame" />
   <addaction name="actionVCRPreviousFrame" />
   <addaction name="actionVCRNextFrame" />
   <addaction name="actionVCRLastFrame" />
  </widget>
  <widget class="QToolBar" name="selectionToolbar" >
   <property name="windowTitle" >
    <string>Selection Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionMoveMode" />
   <addaction name="actionSelectionMode" />
  </widget>
  <widget class="QToolBar" name="undoRedoToolbar" >
   <property name="windowTitle" >
    <string>Undo/Redo Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionEditUndo" />
   <addaction name="actionEditRedo" />
  </widget>
  <widget class="QToolBar" name="variableToolbar" >
   <property name="windowTitle" >
    <string>Active Variable Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="customFilterToolbar" >
   <property name="windowTitle" >
    <string>Custom Filter Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <action name="actionFileNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionHelpHelp" >
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionServerConnect" >
   <property name="text" >
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionServerDisconnect" >
   <property name="text" >
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionFileSaveScreenshot" >
   <property name="text" >
    <string>Save Screenshot</string>
   </property>
  </action>
  <action name="actionFileSaveAnimation" >
   <property name="text" >
    <string>Save Animation</string>
   </property>
  </action>
  <action name="actionVCRPreviousFrame" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Previous Frame</string>
   </property>
  </action>
  <action name="actionVCRFirstFrame" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>First Frame</string>
   </property>
  </action>
  <action name="actionVCRNextFrame" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Next Frame</string>
   </property>
  </action>
  <action name="actionVCRLastFrame" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Last Frame</string>
   </property>
  </action>
  <action name="actionFileSaveData" >
   <property name="text" >
    <string>Save Data</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileLoadServerState" >
   <property name="text" >
    <string>&amp;Load Server State</string>
   </property>
  </action>
  <action name="actionFileSaveServerState" >
   <property name="text" >
    <string>&amp;Save Server State</string>
   </property>
  </action>
  <action name="actionHelpEnableTooltips" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Enable Tooltips</string>
   </property>
  </action>
  <action name="actionMoveMode" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Move Mode</string>
   </property>
  </action>
  <action name="actionSelectionMode" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Selection Mode</string>
   </property>
  </action>
  <action name="actionEditUndo" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEditCameraUndo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Camera Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEditCameraRedo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Camera Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToolsDumpWidgetNames" >
   <property name="text" >
    <string>&amp;Dump Widget Names</string>
   </property>
  </action>
  <action name="actionToolsRecordTest" >
   <property name="text" >
    <string>&amp;Record Test</string>
   </property>
  </action>
  <action name="actionToolsRecordTestScreenshot" >
   <property name="text" >
    <string>Record &amp;Test Screenshot</string>
   </property>
  </action>
  <action name="actionToolsPlayTest" >
   <property name="text" >
    <string>&amp;Play Test</string>
   </property>
  </action>
  <action name="actionToolsPythonShell" >
   <property name="text" >
    <string>Python &amp;Shell</string>
   </property>
  </action>
  <action name="actionToolsCreateCustomFilter" >
   <property name="text" >
    <string>&amp;Create Custom Filter ...</string>
   </property>
  </action>
  <action name="actionToolsManageCustomFilters" >
   <property name="text" >
    <string>&amp;Manage Custom Filters ...</string>
   </property>
  </action>
  <action name="actionFoo" >
   <property name="text" >
    <string>Foo</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
